source/whats-new.txt

```diff
@@ -29,6 +30,21 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.1:
+
+What's New in 5.1
+-----------------
+
+New features of the 5.1 {+driver-short+} release include:
+
+- The driver now supports automatic serialization of JavaScript ``bigint`` to
+  ``BSON.Long``. It also supports deserializing of ``BSON.Long`` values returned
+  from the server to ``bigint`` values when the ``useBigInt64`` flag is passed
```

Comment:
Suggestion

Minor grammar thing here - "It also supports deserializing of..." seems awkward to me. I think it would read better as either "It also supports deserializing..." (remove the of) or "It also supports the deserialization of..."